movie-project/src/components/MovieItem.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { Card, Button, Badge } from "react-bootstrap";
+import MovieOverview from './MovieOverview'
+let imageBaseUrl = "https://image.tmdb.org/t/p/w500";
+
+export default function MovieItem(props) {
```

Comment:
extract the props like
```suggestion
export default function MovieItem({ movie }) {
```